althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -176,6 +200,49 @@ impl dyn KernelInterface {
         Ok(())
     }
 
+    pub fn set_ipv6_route_to_tunnel(&self, exit_ip: IpAddr) -> Result<(), Error> {
+        // For now gateway is set to fe80:: + exit ip, and this is ensured to be the case on the exit also
+        if let IpAddr::V6(ip_addr) = exit_ip {
+            let seg = ip_addr.segments();
+            let gw = IpAddr::V6(Ipv6Addr::new(
+                0xfe80, seg[1], seg[2], seg[3], seg[4], seg[5], seg[6], seg[7],
```

Comment:
this should use the function https://github.com/althea-net/althea_rs/blob/master/althea_kernel_interface/src/open_tunnel.rs#L6